crypto/stark-verifier-ethereum/test/merkle_test.ts

```diff
@@ -0,0 +1,76 @@
+import { deployContract, solidity } from "ethereum-waffle";
+import { waffle } from "@nomiclabs/buidler";
+import chai from "chai";
+import {tx_to_events} from './test_utils';
+
+import MerkleVerifierTestArtifact from "../artifacts/MerkleVerifierTest.json";
+import { MerkleVerifierTest } from "../typechain/MerkleVerifierTest";
+
+chai.use(solidity);
+const { expect } = chai;
+
+describe("Public coin testing", () => {
+  let merkle_contract: any;
+  const init_hex = "0x0123456789abcded";
+
+  const provider = waffle.provider;
+  let [wallet] = provider.getWallets();
+
+  before(async function() {
+    merkle_contract = await deployContract(wallet, MerkleVerifierTestArtifact) as MerkleVerifierTest;
+  });
```

Comment:
Can we add a test where clamed_data is empty? What should the correct behaviour be?